client/containers/ImportExport.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { Button } from 'react-bootstrap';
+import BlankState from '../components/Dashboard/BlankState';
+
+import { importExportActions } from '../actions';
+import { Error, LoadingPanel, Json } from '../components/Dashboard';
+
+import GroupsIcon from '../components/Dashboard/icons/GroupsIcon';
+
+class ImportExportContainer extends Component {
+  componentWillMount() {
+    this.props.exportConfig();
```

Comment:
I think you probably want to use [componentDidMount](https://facebook.github.io/react/docs/component-specs.html#mounting-componentdidmount) instead of componentWillMount, because you're fetching data here.